packages/runtime/src/parser/parser.ts

```diff
@@ -280,5 +299,9 @@ export async function parser (program: string, interpreter: Interpreter): Promis
       opCodeList.push(opcodeFromSentence(words, counter, interpreter));
     }
   }
+
+  if (interpreter.gas > LogicSigMaxCost) {
```

Comment:
let's add a documentation about when are we calculating gas, at suitable place.

Reply:
Alright